src/js/big-play-button.js

```diff
@@ -8,16 +11,20 @@
  * @class
  * @constructor
  */
-vjs.BigPlayButton = vjs.Button.extend();
+var BigPlayButton = Button.extend();
```

Comment:
I originally had `let` for pretty much everything, but for some reason, JS Hint was complaining about exporting them.